osbs/api.py

```diff
@@ -602,11 +612,13 @@ def create_worker_build(self, **kwargs):
         for required in ('platform', 'release', 'arrangement_version'):
             if not kwargs.get(required):
                 missing.add(required)
-
         if missing:
             raise ValueError("Worker build missing required parameters: %s" %
                              missing)
 
+        if kwargs.get('platforms'):
```

Comment:
I agree with this change, but it seems unrelated to node selectors. Can we split up into different commits?